crypto/ct/ct_sct_ctx.c

```diff
@@ -111,7 +111,7 @@ static int ct_x509_get_ext(X509 *cert, int nid, int *is_duplicated)
  * AKID from the presigner certificate, if necessary.
  * Returns 1 on success, 0 otherwise.
  */
-static int ct_x509_cert_fixup(X509 *cert, X509 *presigner)
+__owur static int ct_x509_cert_fixup(X509 *cert, X509 *presigner)
```

Comment:
No, don’t bother with that.  If/when we're able to fix the ASN1 stuff, we can move things upstream to callers.